package.json

```diff
@@ -104,25 +105,29 @@
     "resq": "1.10.2",
     "serve": "13.0.2",
     "sharp": "0.30.7",
+    "terser": "5.15.1",
     "ts-jest": "28.0.2",
     "ts-node": "10.8.1",
-    "typescript": "4.6.4"
+    "typescript": "4.6.4",
+    "vite": "3.1.8"
   },
   "dependencies": {
```

Comment:
linkurious/rest-client & linkurious/ogma-linkurious-parser should be:
- in the same version
- pinned
- in 3.1.12 (latest LKE)

Reply:
done, put 3.1.12